encoding/legacy-mb-schinese/gb18030/gb18030-decoder.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
```

Comment:
Maybe make this a `.any.js` test to reduce boilerplate and increase coverage to workers?

Reply:
This test matches the pattern established by many other tests in the legacy-mb-* directories of not testing in workers and non-workers.  There are other tests that verify that encodings in worker contexts behave as they ought. There are already over a hundred thousand tests in these directories, and they take too long in debug builds that we don't even run them in debug builds in our continuous integration.  I'd rather not break with the pattern established by other multibyte encoding tests and add worker tests here unless another reviewer also thinks we should.

Reply:
Fair points re: consistency and performance. (We could bulk convert files to `.window.js` but that's not in scope for this PR)